pkg/storage/couchdb/errors.go

```diff
@@ -9,6 +9,7 @@ import "errors"
 
 const (
 	// Error messages we return
+	failToPingCouchDB                                  = "failure while pinging couchDB:"
```

Comment:
Is there any reason you are not using the same format as other errors "msg: %w"?

Reply:
I wanted to check if the error contains this string in unit test

Reply:
would it be better to use the same format and just copy the string without the "%w" to compare errors in unit test?

Reply:
Yes, it would be more consistent with other tests. You can use contains to check for "failure while pinging couchDB"

Reply:
fixed

Reply:
Also, keep in mind how other code will check for this error... the error message is CouchDB-specific. You want to make sure you avoid a situation where some code in another project is dealing with a "general provider", but then you check for a specific error cause by one of the provider implementations.